source/jsonify.d.ts

```diff
@@ -1,12 +1,32 @@
 import type {JsonPrimitive, JsonValue} from './basic';
 import type {EmptyObject} from './empty-object';
-import type {Merge} from './merge';
 import type {NegativeInfinity, PositiveInfinity} from './numeric';
+import type {Simplify} from './simplify';
 import type {TypedArray} from './typed-array';
 
+/*
+ * `any` is the only type that can let you equate `0` with `1`
+ * See https://stackoverflow.com/a/49928360/1490091
+ */
+type IsAny<T> = 0 extends 1 & T ? true : false;
```

Comment:
Can you move this to internal.d.ts? We keep reusable types there.

Reply:
done